subsys/net/lib/zperf/zperf_shell.c

```diff
@@ -163,9 +163,7 @@ static int parse_ipv6_addr(const struct shell *sh, char *host, char *port,
 
 	ret = net_addr_pton(AF_INET6, host, &addr->sin6_addr);
 	if (ret < 0) {
-		shell_fprintf(sh, SHELL_WARNING,
-			      "Invalid IPv6 address %s\n", host);
-		return -EINVAL;
+		return -EFAULT;
```

Comment:
I am not sure about the errno, it is more related to memory address failures.
Would for example `EDESTADDRREQ` (Destination address required) be more appropriate here?

Reply:
yeah I considered that one EDESTADDREQ too, I honestly wasn't sure which errno to use, so completely fine to change it